Spigot-Server-Patches/0380-Handle-bad-chunks-more-gracefully.patch

```diff
@@ -29,20 +29,23 @@ index c53518a47..6f34d8aea 100644
 +
 +                try {
 +                    oversizedData = regionfile.getOversizedData(chunkCoordinate.x, chunkCoordinate.z);
-+                } catch (Exception ex) {}
++                } catch (Exception ex) {
++                    MinecraftServer.LOGGER.error("Failed to read OVERSIZED chunk data for " + chunkCoordinate + " in regionfile " + regionfile.file.toString() + ", ignoring OVERSIZED data", ex);
```

Comment:
`toString()` in `regionfile.file.toString()` is unnecessary, no?

Reply:
I've got a habit of explicitly toString()ing objects in concatenation